### PR TITLE
[FIX] Missing strings in all languages

### PR DESCRIPTION
**Pull request by rafaelks**
@RocketChat/ios

### CODE REVIEW

File: Rocket.Chat/cs.lproj/Localizable.strings [hunk 1]
```diff
@@ -141,3 +153,9 @@
 "chat.messages.list.empty" = "Nebyly nalezeny žádné zprávy";
 "chat.messages.pinned.list.title" = "Připnuté zprávy";
 "chat.messages.starred.list.title" = "Zprávy s hvězdičkou";
+
+// Settings
+"myaccount.settings.title" = "Settings";
+"myaccount.settings.contactus" = "Contact us";
+"myaccount.settings.license" = "License";
+"myaccount.settings.version" = "Version: %@ (%@)";
```
Comment (rafaelks) [inline Rocket.Chat/cs.lproj/Localizable.strings]: @flashbackcz I know it sucks, but could you help us translating this new strings? We're working to integrate with LingoHub soon.

File: Rocket.Chat/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -134,11 +154,8 @@
...
-"new_room.cell.read_only.title" = "Schreibgeschützter Kanal";
-"new_room.cell.read_only.description" = "Nur Moderatoren oder Admins können in schreibgeschützten Kanälen schreiben";
-"new_room.group.channel.name" = "Channel Name";
-"new_room.cell.channel_name.title" = "Channel Name";
-"new_room.group.channel.footer" = "Der Name muss komplett in Kleinbuchstaben und kürzer als 22 Zeichen sein.";
+// Settings
+"myaccount.settings.title" = "Settings";
+"myaccount.settings.contactus" = "Contact us";
+"myaccount.settings.license" = "License";
+"myaccount.settings.version" = "Version: %@ (%@)";
```
Comment (rafaelks) [inline Rocket.Chat/de.lproj/Localizable.strings]: @TheReal1604 I know it sucks, but could you help us translating this new strings? We're working to integrate with LingoHub soon.

File: Rocket.Chat/fr.lproj/Localizable.strings [hunk 3]
```diff
@@ -132,3 +154,8 @@
 "chat.messages.pinned.list.title" = "Messages épinglés";
 "chat.messages.starred.list.title" = "Messages favoris";
 
+// Settings
+"myaccount.settings.title" = "Settings";
+"myaccount.settings.contactus" = "Contact us";
+"myaccount.settings.license" = "License";
+"myaccount.settings.version" = "Version: %@ (%@)";
```
Comment (rafaelks) [inline Rocket.Chat/fr.lproj/Localizable.strings]: @sowenjub I know it sucks, but could you help us translating this new strings? We're working to integrate with LingoHub soon.

File: Rocket.Chat/cs.lproj/Localizable.strings (hunk 1 above)
Comment (flashbackcz) [reply Rocket.Chat/cs.lproj/Localizable.strings]: Tomorrow I will do it (;

I have to admit that LingoHub is cool, there I translated some strings to czech too.

File: Rocket.Chat/de.lproj/Localizable.strings (hunk 2 above)
Comment (TheReal1604) [reply Rocket.Chat/de.lproj/Localizable.strings]: Sure! I take a look...

**Pull request merged**